lib/lighthouse.js

```diff
@@ -35,7 +35,7 @@ const LIGHTHOUSE_OPTS = {
 function run() {
   const configName = getConfigName();
   const config = getConfig(configName);
-  const isTerser = getIsTerser();
+  const isPretty = process.env.A11Y_PRETTY === true || process.env.A11Y_PRETTY === 'true';
```

Comment:
Why does it need both?

Reply:
If it gets called from the package script, it'll be a string.

Reply:
Just use a string everywhere then? :)

Reply:
lol ok- it means putting a `thing = 'true'` right next to an `otherThing = true` which kinda crimped my sense of orderliness 😂